### PR TITLE
Initial FGD to markdown generator

### PR DESCRIPTION
**Pull request by braem**
Hooks into the FGD compile process to allow for generating markdown.

Ideally we will host this on some static site with additional, manually-written usage documentation appended.

### CODE REVIEW

**Comment by TeamSpen210** [comment]
Very nice! Not sure though if it'd be better to generate these on the tagged database, then add the little "only in X" icons. With a preset list of tags → icons, it'd be possible to match up.

**Comment by braem** [comment]
> Very nice! Not sure though if it'd be better to generate these on the tagged database, then add the little "only in X" icons. With a preset list of tags → icons, it'd be possible to match up.

I was thinking something similar yeah, where you can see what entities/kvs/etc are in which game. Ended up scrapping it as it is more complicated and mappers don't tend to stray from one particular game. Someone mapping for momentum for instance most likely only wants to see exactly what is in momentum.

I think from a standpoint of your project, it would be quite cool and useful to be able to see what is in a multitude of different games like that though - coming from my stance anyways where I frequently have to search the FGDs of older games to determine their structure for fixing bugs.

File: fgd/bases/CombineBallSpawners.fgd [hunk 1]
```diff
@@ -12,7 +12,7 @@
 	maxspeed(float) : "Max ball speed" : 600.0 : "The maximum speed of balls that fly in the spawner"
 
 	ballradius(float) : "Ball radius" : 20.0 : "The radius of the energy balls."
-	balltype(choices) : "Ball Type" : "Combine Energy Ball 1" =
+	balltype(choices) : "Ball Type" : 0 =
```
Comment (AWildErin) [inline fgd/bases/CombineBallSpawners.fgd:15]: How come this line was changed?

**Review by AWildErin — APPROVED** [review]
Seems fine to me from a quick glimpse. Only had one comment, but another dev should look this over to make sure the python code is fine

File: fgd/bases/CombineBallSpawners.fgd (hunk 1 above)
Comment (AWildErin) [reply fgd/bases/CombineBallSpawners.fgd:15]: Just saw the commit attached to this so ignore me 🗿

File: .github/workflows/build_md.yml [hunk 2]
```diff
@@ -0,0 +1,47 @@
+name: Markdown Build
+on:
+  pull_request:
+    branches:
+      - master
```
Comment (Gocnak) [inline .github/workflows/build_md.yml:5]: Should this run on every pull request? Seems more like a manual run thing as well, like the release is.

**Review by Gocnak — CHANGES_REQUESTED** [review]

File: .github/workflows/build_md.yml (hunk 2 above)
Comment (braem) [reply .github/workflows/build_md.yml:5]: My rationale is that we'd like to know whether it's able to convert to markdown, but it does use very much the same code as a FGD compile so I suppose we don't need to run it outside of releases.

Comment (braem) [reply .github/workflows/build_md.yml]: Switched to over to be apart of the release CI instead of build.

**Review by Gocnak — APPROVED** [review]

**Pull request merged**